htdocs/core/class/html.formsetup.class.php

```diff
@@ -215,7 +215,7 @@ public function generateTableOutput($editMode = false)
 			$out = '<table class="noborder centpercent">';
 			$out .= '<thead>';
 			$out .= '<tr class="liste_titre">';
-			$out .= '	<td class="titlefield">' . $this->langs->trans("Parameter") . '</td>';
```

Comment:
i remove this css class because it generate a design error for setup table (it fix title col to 25% of width )